core/ir/instr_shared.c

```diff
@@ -3101,6 +3105,7 @@ instr_convert_to_isa_regdeps(void *drcontext, instr_t *instr_real_isa,
         for (uint reg = 0; reg < REGDEPS_MAX_NUM_REGS; ++reg) {
             if (dst_reg_used[reg]) {
                 opnd_t dst_opnd = opnd_create_reg((reg_id_t)reg);
+                opnd_set_size(&dst_opnd, max_opnd_size);
```

Comment:
This is more to make instr_same() (which checks the register size) work.
Now that we are mapping to virtual registers, we might end up having a DR_REG_V whose value is, for example, one of the DR_REG_XMM reserved ones, so calling reg_get_size() on it would raise the ASSERT: "reg_get_size: invalid register", since it wouldn't match any of the if-else cases.
However, it never really made sense to query the size of a virtual register, even before.
Anyway, added comment.
Done.